metrics.rst

```diff
@@ -5,6 +5,11 @@ Characterization Metric Reports
 Starting from Summer 2015, administrative ("cycle") releases are accompanied by a measurements report characterizing the current performance.
 Metrics included in these reports are expected to increase in number and sophistication at subsequent releases.
 
+- :ref:`Release 25.0.0 <release-v25-0-0>`: `DMTR-392 <https://ls.st/DMTR-392>`_
+- :ref:`Release 24.1.0 <release-v24-1-0>`: `DMTR-391 <https://ls.st/DMTR-391>`_
```

Comment:
With updating release notes for the patch releases v24.1.3, v25.0.3 and v26.0.0 
this is better done in the release branches and cherry picked to main.
I will take are of the changes in this PR.